lib/puppetserver/ca/config/puppet.rb

```diff
@@ -130,7 +131,6 @@ def resolve_settings(overrides = {})
             :hostcert => '$certdir/$certname.pem',
             :hostprivkey => '$privatekeydir/$certname.pem',
             :hostpubkey => '$publickeydir/$certname.pem',
-            :publickeydir => '$ssldir/public_keys',
```

Comment:
This isn't related to my PR.  Just noticed it was a duplicate (it's in base-defaults above)

Reply:
Good catch.